src/Surface.zig

```diff
@@ -675,6 +675,22 @@ pub fn init(
             .set_title,
             .{ .title = title },
         );
+    } else if (config.@"initial-command") |initial_command| {
+        switch (initial_command) {
+            // If a user specifies a command with `-e` it is appropriate to set the title
+            // as argv[0]
+            .direct => {
+                _ = try rt_app.performAction(
+                    .{ .surface = self },
+                    .set_title,
+                    .{ .title = initial_command.direct[0] },
+                );
```

Comment:
The value of `direct` should be captured by the switch statement rather than using `initial_command.direct` to access the value.